AccessCheckoutSDK/AccessCheckoutSDKTests/validation/card/CardValidatorTests.swift

```diff
@@ -901,13 +651,10 @@ class CardValidatorTests: XCTestCase {
         XCTAssertTrue(valid.partial)
         XCTAssertFalse(valid.complete)
     }
-    
+
     func testValidateExpiryMonth_alpha() {
```

Comment:
As a side note - We'll have to review all tests around validation cause some of them like this one just test regular expressions set up in the test itself rather than testing just production code. Nothing to worry about now as long as our build passes